modules/jwplayerTargeting.js

```diff
@@ -0,0 +1,167 @@
+import { config } from '../src/config.js';
+import { ajaxBuilder } from '../src/ajax.js';
+import { logError, isPlainObject } from '../src/utils.js';
+import { getGlobal } from '../src/prebidGlobal.js';
+import { module } from '../src/hook.js';
+
+const segCache = {};
+let requestCount = 0;
+let requestTimeout;
+let resumeBidRequest;
+
+/*
+Prebid auctions timeout at 200ms.
+ */
+let feedFetchTimeout = 150;
+
+config.getConfig('jwTargeting', config => {
+  const targeting = config.jwTargeting;
+  // fetch media ids
+  fetchTargetingInformation(targeting);
+
+  const prefetchTimeout = targeting.prefetchTimeout;
+  if (prefetchTimeout) {
+    // prefetch timeout supersedes our default and our adjustment for bidderTimeout.
+    feedFetchTimeout = prefetchTimeout;
+    return;
+  }
+
+  const timeout = config.bidderTimeout; 
+  if (timeout < 200) {
```

Comment:
What if `timeout * 3 / 4` is greater than 200? I don't think this is the right solution

Reply:
if timeout is less than 200 then how could multiplying it by 0.75  make it bigger ?